test.js

```diff
@@ -262,6 +262,35 @@ test('Expose custom status route', t => {
   })
 })
 
+test('Expose status route with additional route options', t => {
```

Comment:
Could you add a test where the `exposeStatusRoute` doesn't have the `route` parameter to test the default?

Reply:
Test added